.env.sample

```diff
@@ -1,16 +1,21 @@
+# Server config
 PORT=8000
+CORS_WHITELIST=https://anj.aragon.org
```

Comment:
Could we add `court.aragon.org` to this list? I think it should just be a `,https://court.aragon.org` but configuring arrays via `.env`s always take a bit to figure out for me.